TACC/Core-CMS
# Add ipython to django shell

## Overview

Adds the ipython package to the environment, via `poetry add ipython`

By default, django will use ipython for the django shell if it is installed:
https://docs.djangoproject.com/en/2.2/ref/django-admin/#shell

## Testing

1. Build image: `make build`
2. Start it up: `make start`
3. Bring up django shell from within container:
  ```
docker exec -it core_cms bash
./manage.py shell
```
4. Confirm ipython is installed.

## UI

**Before (without ipython):**
![Screen Shot 2022-08-16 at 1 51 32 PM](https://user-images.githubusercontent.com/20326896/184957631-651a25c5-1d99-47b9-ad23-d3636dc4ffd4.png)

**After (with ipython):**
![Screen Shot 2022-08-16 at 1 57 54 PM](https://user-images.githubusercontent.com/20326896/184959829-62a982ca-a853-4d6d-a7d8-255b5157320e.png)
